### ISSUE TITLE
Trying to train a model following the commands in the readme, getting a few errors

### ISSUE THREAD

**Issue by patbohn** [uedaLabR/nanoDoc#3]
Dear Dr. Ueda, 

I am intrigued to use your software for the detection of RNA modifications in our experiments. Would we benefit from training our own model, or are the currently trained weights already detecting all modifications ("outliers") regardless of their nature? 

In addition, trying to get it to run, I've collected some feedback: 
When **running formatfile**, it appears that the most time is taken up by a single-threaded workload coming after "mark parquet", not sure if this is intended or whether threading it may increase speed by a lot. 

When **running make5mer**, the 5mer_train output directory is not automatically created, maybe an "os.makedirs()" could circumvent creating it manually? 

A second thing I noticed when running make5mer is that it looks for a parquet file named "algined"X.pq, while the files were previously named only X.pq, so I had to rename it to be detected. 

Now when attempting to run **train_cnn** I ran into more severe issues, which I believe may be _partially due to my GPU (RTX 3070), that required CUDA11, cuDNN8 and tensorflow 2.4 to be installed for it to work_. After getting TF to run, I noticed that there is an unlabeled plot being generated, and I needed to close this for the process to continue. 

Also, (presumably due to feeding in our own data of a sequence that may not contain all 1024 5-mers) I received an "index out of range" error from keras to_categorical, as the labels start at 1, but keras expects them to start at 0. Manually setting number of labels to 1024 fixed this error for now (and also a later error that had occurred due to category size mismatch). 

Where I am now getting stuck however is that I can only perform training for 25 epochs due to RAM limitations, and it seems that the weights stored there are not directly compatible with the next step, train_doc (although I am getting accuracy > 0.5). 

Is there a simple way to reduce the RAM requirements (have 32 GB + 20 GB swap)? 

Best wishes, 
Patrick Bohn

**Comment by uedaLabR** [comment]
Hi Patrick

Thank you for your detailed comment.
This program could detect all outliers with currently trained weight, in theory, at least.  
Training your own model could potentially increase the  sensibility against particular modifications,
however, I could not get better result in my little trials.  

running formatfile> I will check and  make it into multi threaded if possible
running make5mer > Thanks. I fixed it.
"algined"X.pq,... Xpq > I also fixed it

I have never tried but newer version of TF and CUDA and cuDNN , I dont know your unlabeled plot is something to do with
my code but I will check about it.
I recommend using training sample with  some reference length which cover 1024 5-mer, but sorry that you had to workaround.

I thought the weight is compatible with next step. I will check about this too.

I will update if I tested by myself.

Thanks
Hiroki
